hr_timesheet_sheet_period/models/res_company.py

```diff
@@ -3,14 +3,12 @@
 
 from odoo import fields, models
 
-SHEET_RANGE_PAYROLL_PERIOD = 100
```

Comment:
please keep the constant just make it a string :) otherwise this magic "100" is everywhere

Reply:
@alexey-pelykh done :)